internal/widget/reddit.go

```diff
@@ -19,6 +19,7 @@ type Reddit struct {
 	CommentsUrlTemplate string          `yaml:"comments-url-template"`
 	Limit               int             `yaml:"limit"`
 	CollapseAfter       int             `yaml:"collapse-after"`
+	CustomEndpoint      string          `yaml:"custom-endpoint"`
```

Comment:
```suggestion
	CustomEndpoint      string          `yaml:"request-url-template"`
```

Just to keep it consistent with the rest of the parameters that provide similar functionality.